.eslintcache

```diff
@@ -1 +1 @@
nghakim/14-2nd-SHABANG-frontend/14-2nd-SHABANG-frontend/src/Pages/Apartment/components/Map/MapContainer.js":"3","/Users/donghakim/14-2nd-SHABANG-frontend/14-2nd-SHABANG-frontend/src/index.js":"4","/Users/donghakim/14-2nd-SHABANG-frontend/14-2nd-SHABANG-frontend/src/Pages/LogIn/LogIn.js":"5","/Users/donghakim/14-2nd-SHABANG-frontend/14-2nd-SHABANG-frontend/src/Styles/GlobalStyles.js":"6"},{"size":108,"mtime":1606755108482,"results":"7","hashOfConfig":"8"},{"size":489,"mtime":1607017930313,"results":"9","hashOfConfig":"8"},{"size":7099,"mtime":1607066130241,"results":"10","hashOfConfig":"8"},{"size":277,"mtime":1607025778101,"results":"11","hashOfConfig":"8"},{"size":106,"mtime":1606755108481,"results":"12","hashOfConfig":"8"},{"size":523,"mtime":1606908097905,"results":"13","hashOfConfig":"8"},{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"152w81i",{"filePath":"16","messages":"17","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"18","messages":"19","errorCount":0,"warningCount":3,"fixableErrorCount":0,"fixableWarningCount":0,"source":null},{"filePath":"20","messages":"21","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"22","messages":"23","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"24","messages":"25","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/donghakim/14-2nd-SHABANG-frontend/14-2nd-SHABANG-frontend/src/Pages/OneRoom/OneRoom.js",[],"/Users/donghakim/14-2nd-SHABANG-frontend/14-2nd-SHABANG-frontend/src/Pages/Apartment/Apartment.js",[],"/Users/donghakim/14-2nd-SHABANG-frontend/14-2nd-SHABANG-frontend/src/Pages/Apartment/components/Map/MapContainer.js",["26","27","28"],"/Users/donghakim/14-2nd-SHABANG-frontend/14-2nd-SHABANG-frontend/src/index.js",[],"/Users/donghakim/14-2nd-SHABANG-frontend/14-2nd-SHABANG-frontend/src/Pages/LogIn/LogIn.js",[],"/Users/donghakim/14-2nd-SHABANG-frontend/14-2nd-SHABANG-frontend/src/Styles/GlobalStyles.js",[],{"ruleId":"29","severity":1,"message":"30","line":3,"column":38,"nodeType":"31","messageId":"32","endLine":3,"endColumn":47},{"ruleId":"29","severity":1,"message":"33","line":5,"column":10,"nodeType":"31","messageId":"32","endLine":5,"endColumn":27},{"ruleId":"34","severity":1,"message":"35","line":29,"column":6,"nodeType":"36","endLine":29,"endColumn":28,"suggestions":"37"},"no-unused-vars","'Component' is defined but never used.","Identifier","unusedVar","'KAKAO_MAP_API_KEY' is defined but never used.","react-hooks/exhaustive-deps","React Hook useEffect has missing dependencies: 'getMap' and 'getZoomedMap'. Either include them or remove the dependency array.","ArrayExpression",["38"],{"desc":"39","fix":"40"},"Update the dependencies array to be: [getMap, getZoomedMap, placeData, zoomLevel]",{"range":"41","text":"42"},[725,747],"[getMap, getZoomedMap, placeData, zoomLevel]"]
```

Comment:
You can untrack a specific file with `git rm --cached .eslintcache`

Reply:
I did it.